python/snewpy/scripts/snewpy_to_snewpdag.py

```diff
@@ -0,0 +1,67 @@
+
+from argparse import ArgumentParser
+from snewpy import to_snowglobes
+from snewpy import run_snowglobes
+from snewpy import from_snowglobes
+import tarfile
+import numpy as np
+import os
+from astropy.io import ascii
+from astropy import units as u
+import matplotlib.pyplot as plt
```

Comment:
```suggestion
from snewpy import to_snowglobes, run_snowglobes, from_snowglobes
import numpy as np
from astropy import units as u
```
removing unused imports